dist/image-sequencer.js

```diff
@@ -47721,6 +47721,25 @@ ImageSequencer = function ImageSequencer(options) {
     else modulesdata = modules[name][1];
     return modulesdata;
   }
+
+  function toString(step_name,options){
+    if(step_name){
+      return `${step_name}(${Object.keys(options).map(key=>key + ':' + options[key]).join(',')})`
+    }
+    else{
```

Comment:
Same on formatting here - i prefer `} else {` style, not on independent lines. Thanks!